kinto/core/resource/viewset.py

```diff
@@ -120,13 +120,13 @@ def get_view_arguments(self, endpoint_type, resource_cls, method):
         args.update(**endpoint_args)
 
         if method.lower() in map(str.lower, self.validate_schema_for):
-            schema = PartialSchema()
+            schema = RequestSchema()
             record_schema = self.get_record_schema(resource_cls, method)
             record_schema.name = 'body'
             schema.add(record_schema)
             args['schema'] = schema
         else:
-            args['schema'] = SimpleSchema()
+            args['schema'] = RequestSchema()
```

Comment:
Well, I've got why `StrictSchema` and `SimpleSchema`were needed by `get_record_schema`but I actually didn't understand why we needed `PartialSchema` and `SimpleSchema` on this one, aren't we just setting the request schema here (in contrary to the body schema)? 

Also, this didn't break any tests and the API behavior looks ok, so I think it's ok to keep the same behavior to both.